packages/back-end/src/models/ExperimentModel.ts

```diff
@@ -399,6 +400,10 @@ export async function getAllExperiments(
     query.type = "multi-armed-bandit";
   } else if (type === "standard") {
     query.type = { $in: ["standard", null] };
+  } else if (type === "holdout") {
+    query.type = "holdout";
+  } else if (!type) {
+    query.type = { $ne: "holdout" };
```

Comment:
This is covering up a pre-existing bug.  Ok to leave, but we should fix separately